src/Microsoft.SqlTools.ServiceLayer/Connection/ConnectionType.cs

```diff
@@ -0,0 +1,19 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+namespace Microsoft.SqlTools.ServiceLayer.Connection
```

Comment:
I didn't see the original comment, why was doing this as an enum not beneficial?